hack/utilities.sh

```diff
@@ -0,0 +1,20 @@
+#
+# Library of useful utilities for CI purposes.
+#
+
+readonly RED='\033[0;31m'
+readonly GREEN='\033[0;32m'
+readonly INVERTED='\033[7m'
+readonly NC='\033[0m' # No Color
```

Comment:
Do we need to use colors in terminal?

Reply:
This was done imitating the output format from all other scripts we have in kyma and on prow. Helps having a nicer output and colors help driving the attention to the important stuff.